crates/cxx-qt-lib-headers/Cargo.toml

```diff
@@ -13,5 +13,8 @@ repository.workspace = true
 
 [features]
 default = []
-qt_gui = []
-qt_qml = []
+qt_gui = ["cxx-qt-build/qt_gui"]
```

Comment:
The `cxx-qt-build/qt_gui` feature isn't necessary anymore, no?

Reply:
Done :-)